### PR TITLE
Update Drools results for 7.73.0.Final.

### PR DESCRIPTION
**Pull request by baldimir**

### CODE REVIEW

File: TestResults/Drools/7.73.0.Final/tck_results.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Wed Oct 05 14:21:09 CEST 2022
+last.update=2022-10-05
+product.version=7.73.0.Final
```
Comment (tarilabs) [inline TestResults/Drools/7.73.0.Final/tck_results.properties:3]:
```suggestion
product.version=7.73.0.Final
product.url=http\://www.drools.org
vendor.url=http\://www.redhat.com
product.comment=Drools provides a compliance level 3 runtime DMN engine
vendor.name=Red Hat
instructions.url=https\://github.com/dmn-tck/tck/tree/master/runners/dmn-tck-runner-drools/README.md
product.name=Drools
```

**Review by tarilabs — CHANGES_REQUESTED** [review]
You definitely need to maintain a vendor supplied property attributes. See details below.

I believe if you just run the runner on version bump (without deleting files first), it would maintain the properties at the new version. IIRC.

**Comment by baldimir** [comment]
Thank you for the suggestion.

**Comment by baldimir** [comment]
@tarilabs Is it ok now please? Can I merge?

**Review by tarilabs — APPROVED** [review]
I do believe 7b00e4f implemented the required changes, yes

**Pull request merged**